### DISCUSSION TITLE
v1.0.0

### DISCUSSION THREAD

**Discussion by nabaztag2018** [nabaztag2018/pynab#393] [General]
**4 years later, the 1.0.0!**

Thank you to all the contributors!

Amon other things, this 1.0.0 contains:
- A driver for the new NFC board by (who else) @pguyot 
- Lost of enhancements by @pguyot  and @f-laurens 
- Simplified wifi setup by@davesteele
- Interface with[ If This Then That](https://www.ifttt.com) by @omevel 

⚠️ As of June, 14th 2022 : only the Raspian image has ben tested but that does not mean the other images do not work! Please file an issue if you find something weird ⚠️

##### if you want all the gory details 👉
* Fix location of 'It's only 4 o-clock, what a very very long day' (fix… by @pguyot in https://github.com/nabaztag2018/pynab/pull/288
* nabweatherd/nabairqualityd: Avoid service restart on exception when fetching data by @f-laurens in https://github.com/nabaztag2018/pynab/pull/291
* Detect today/tomorrow for ASR triggered weather forecasts by @f-laurens in https://github.com/nabaztag2018/pynab/pull/290
* nabweb: Cleanup pages layout by @f-laurens in https://github.com/nabaztag2018/pynab/pull/285
* Add DietPi support by @f-laurens in https://github.com/nabaztag2018/pynab/pull/283
* Docker-based environment: simplified model (2 containers) by @f-laurens in https://github.com/nabaztag2018/pynab/pull/289
* Change Surprise frequency computation, to match original Nabaztag scale by @f-laurens in https://github.com/nabaztag2018/pynab/pull/295
* Make Surprise frequency more natural. by @f-laurens in https://github.com/nabaztag2018/pynab/pull/297
* The Eternal Sunshine of the Nabaztag by @f-laurens in https://github.com/nabaztag2018/pynab/pull/298
* Different base LED pulsing color depending on networking status by @f-laurens in https://github.com/nabaztag2018/pynab/pull/294
* Stop using PiShrink to compress release image by @f-laurens in https://github.com/nabaztag2018/pynab/pull/296
* GitHub  workflows: do not build for DietPi Buster by @f-laurens in https://github.com/nabaztag2018/pynab/pull/307
* Fix typos & links in web pages by @f-laurens in https://github.com/nabaztag2018/pynab/pull/310
* Docker Dockerfile: remove (unneeded) snips-nlu packages from requirements by @f-laurens in https://github.com/nabaztag2018/pynab/pull/309
* nabweatherd: add sound resources for negative temperatures by @f-laurens in https://github.com/nabaztag2018/pynab/pull/306
* Setup: cleanup Linux distro check by @f-laurens in https://github.com/nabaztag2018/pynab/pull/303
* Add support for Debian 11 (Bullseye) by @f-laurens in https://github.com/nabaztag2018/pynab/pull/305
* Update  System Info page links by @f-laurens in https://github.com/nabaztag2018/pynab/pull/318
* Revert to Python 3.7 by @pguyot in https://github.com/nabaztag2018/pynab/pull/335
* Disable SSH by default as this is a security issue by @pguyot in https://github.com/nabaztag2018/pynab/pull/336
* Add comitup on setup by @pguyot in https://github.com/nabaztag2018/pynab/pull/338
* Fix tests on machines with no eth0 by @pguyot in https://github.com/nabaztag2018/pynab/pull/342
* nabiftttd by @omevel in https://github.com/nabaztag2018/pynab/pull/344
* Also install comitup on DietPi by @f-laurens in https://github.com/nabaztag2018/pynab/pull/340
* Cleanup/refresh PyPi requirements by @f-laurens in https://github.com/nabaztag2018/pynab/pull/319
* Initial support for st25r391x by @pguyot in https://github.com/nabaztag2018/pynab/pull/341
* Fix incompatibility with uninitialized T2T by @pguyot in https://github.com/nabaztag2018/pynab/pull/345
* Bump to 0.7.3 to fix bug where incorrect tag was written by @pguyot in https://github.com/nabaztag2018/pynab/pull/346
* Wait for removal of written tag by @pguyot in https://github.com/nabaztag2018/pynab/pull/347
* Reboot to switch between RFID cards by @pguyot in https://github.com/nabaztag2018/pynab/pull/349
* Cleanup services startup by @f-laurens in https://github.com/nabaztag2018/pynab/pull/328
* Fix RFID card detection heuristic by @pguyot in https://github.com/nabaztag2018/pynab/pull/350
* Install Pynab (and drivers) in /opt directory by @f-laurens in https://github.com/nabaztag2018/pynab/pull/334
* Add NFC driver to upgrade page by @pguyot in https://github.com/nabaztag2018/pynab/pull/352
* Start nabd first with start_all by @pguyot in https://github.com/nabaztag2018/pynab/pull/354
* Fix location data type and migration by @pguyot in https://github.com/nabaztag2018/pynab/pull/355
* Update pynfcdev for fix in tag removal logic by @pguyot in https://github.com/nabaztag2018/pynab/pull/357
* Fix translations of upgrade page (broken by 26f80e09) by @pguyot in https://github.com/nabaztag2018/pynab/pull/358
* Sleep and wakeup tags by @pguyot in https://github.com/nabaztag2018/pynab/pull/356
* Fix identification bug by @pguyot in https://github.com/nabaztag2018/pynab/pull/361
* Fix upgrade script and add test upgrade procedure by @pguyot in https://github.com/nabaztag2018/pynab/pull/362
* Fix upgrade script on Pi Zero 2 by @f-laurens in https://github.com/nabaztag2018/pynab/pull/363
* Simplify CI, build less images by @pguyot in https://github.com/nabaztag2018/pynab/pull/359
* Uninstall py-kaldi-asr as it is binary incompatible by @pguyot in https://github.com/nabaztag2018/pynab/pull/364
* Disable st25 driver by @pguyot in https://github.com/nabaztag2018/pynab/pull/365
* Disable chiming for wakeup/sleep tests by @pguyot in https://github.com/nabaztag2018/pynab/pull/367
* Fix obsolete Django gettext reference by @f-laurens in https://github.com/nabaztag2018/pynab/pull/368
* Enable i2c-dev by @pguyot in https://github.com/nabaztag2018/pynab/pull/370
* Fix ST25R reporting in nabweb by @pguyot in https://github.com/nabaztag2018/pynab/pull/372
* Disable IPv6 in chroot environment for running test and installation by @pguyot in https://github.com/nabaztag2018/pynab/pull/369
* Only process tags in idle mode (fix #353, #360) by @pguyot in https://github.com/nabaztag2018/pynab/pull/373
* Fix upgrade and improve automated testing of upgrades by @pguyot in https://github.com/nabaztag2018/pynab/pull/374
* Fix crash when no RFID card is plugged in by @pguyot in https://github.com/nabaztag2018/pynab/pull/375
* Fix yet another nabclockd test by @pguyot in https://github.com/nabaztag2018/pynab/pull/382
* Acknowledge B311 warnings by @pguyot in https://github.com/nabaztag2018/pynab/pull/381
* Stream MP3s over HTTP & HTTPS by @pguyot in https://github.com/nabaztag2018/pynab/pull/380
* Add some types by @pguyot in https://github.com/nabaztag2018/pynab/pull/379
* Docker env: keep requirements.txt in sync by @f-laurens in https://github.com/nabaztag2018/pynab/pull/377
* Disable SSH (fix #351) by @pguyot in https://github.com/nabaztag2018/pynab/pull/376
* Fix mess of upgrade page (fix #388) by @pguyot in https://github.com/nabaztag2018/pynab/pull/390
* Add interactive mode exit tests by @pguyot in https://github.com/nabaztag2018/pynab/pull/391
* Build on tags and trigger by @pguyot in https://github.com/nabaztag2018/pynab/pull/392

**Full Changelog**: https://github.com/nabaztag2018/pynab/compare/v0.9.1...v1.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/nabaztag2018/pynab/releases/tag/v1.0.0'>v1.0.0</a>.</em>

**Comment by f-laurens** [comment]
> As of June, 14th 2022 : only the Raspbian image has ben tested but that does not mean the other images do not work! 

There is an issue with the ears driver on the other images (see https://github.com/nabaztag2018/pynab/issues/394).

**Comment by mehdichaouch** [comment]
Félicitations 🎊 👍

**Comment by noiwid** [comment]
hello est ce que la v1 fonctionne avec la carte TagTagTag commandée via Ulule ? avec la nouvelle carte NFC. 

Merci pour vos retours.
